UIP.py

```diff
@@ -1,10 +1,12 @@
 from scheduler import scheduler
 
 if __name__ == "__main__":
+
   try:
-    print("Press 1 to connect to internet or any other key to use this application in offline mode")
-    offline = int(input()) != 1
+    print("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.\nPress 1 to connect to internet or any other key to use this application in offline mode")

```

Comment:
@vishvanath45 i would agree with @nkprince007  to use two print statement and i would also suggest to move your print statement above the `try` statement.